File: Python Exams/Basic Exam Collection/Basics Online Exam - 9 and 10 March 2019/02. Football Results/02. Football Results.py
win, loss, equal = 0, 0, 0

for _ in range(3):
    host, guest = input().split(':')
    if host > guest:
        win += 1
    elif host < guest:
        loss += 1
    else:
        equal += 1

print(f'Team won {win} games.\nTeam lost {loss} games.\nDrawn games: {equal}')

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 9 and 10 March 2019/02. Football Results/02. Football Results.js
function footballResults(...arg) {
    let [win, lost, equal] = [0, 0, 0]

    for (let i = 0; i < 3; i++) {
        let [host, guest] = arg[i].split(':')
        host > guest ? win++ : host < guest ? lost++ : equal++
    }

    return `Team won ${win} games.\nTeam lost ${lost} games.\nDrawn games: ${equal}`
}

//console.log(footballResults('3:1', '0:2', '0:0'))